File: spring-boot-mongodb/src/main/java/de/larmic/springbootmongodb/database/model/TweetEntity.java
package de.larmic.springbootmongodb.database.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@Document("tweet")
public class TweetEntity {

    @Id
    private UUID id;

    private String message;

    private LocalDateTime createDate = LocalDateTime.now();

    private LocalDateTime lastUpdateDate = LocalDateTime.now();

    public TweetEntity(final String message) {
        this.message = message;
        this.id = UUID.randomUUID();
    }
}

File: spring-boot-postgres-kotlin/src/main/kotlin/de/larmic/properties/database/model/TweetEntity.kt
package de.larmic.properties.database.model

import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "tweet")
class TweetEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "tweet_generator")
    @SequenceGenerator(name = "tweet_generator", sequenceName = "tweet_id_seq", allocationSize = 1)
    var id: Long? = null

    @Column(name = "message", nullable = false)
    var message: String = ""

    @Column(name = "create_date", nullable = false)
    var createDate = LocalDateTime.now()

    @Column(name = "last_update_date", nullable = false)
    var lastUpdateDate = LocalDateTime.now()

    constructor() {
        // required by jpa
    }
}